packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -226,7 +225,28 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
   ScaleGestureRecognizer({
     Object? debugOwner,
     PointerDeviceKind? kind,
-  }) : super(debugOwner: debugOwner, kind: kind);
+    this.dragStartBehavior = DragStartBehavior.down,
+  }) : assert(dragStartBehavior != null),
+    super(debugOwner: debugOwner, kind: kind);
+
+  /// Determines at what point the gesture is calculated from.
+  ///
+  /// When set to [DragStartBehavior.down], the scale is calculated starting
+  /// from the position where the pointer first contacted the screen.
+  ///
+  /// When set to [DragStartBehavior.start], the scale is calculated starting
+  /// from the position where the scale gesture began. The scale gesture may
+  /// begin after the time that the pointer first contacted the screen if there
+  /// are multiple listeners competing for the gesture. In that case, the
+  /// gesture arena waits to determine whether or not the gesture is a scale
+  /// gesture before giving the gesture to this GestureRecognizer. This happens
+  /// in the case of nested GestureDetectors, for example.
+  ///
+  /// For more information about the gesture arena:
+  /// https://flutter.dev/docs/development/ui/advanced/gestures#gesture-disambiguation
+  ///
+  /// Defaults to [DragStartBehavior.down].
```

Comment:
I wrote these docs myself rather than taking anything from DragGestureDetector for now, because those docs seem wrong to me.

https://github.com/flutter/flutter/blob/32ab54029522480cebcf3446b02fdb29ffad6007/packages/flutter/lib/src/gestures/monodrag.dart#L78-L79

I tried this out with nested GestureDetectors and onHorizontalDragStart was called at start, not at down, regardless of what dragStartBehavior was set to.  Am I misunderstanding this?

Reply:
`_checkStart` is only called in `acceptGesture` so the 
>  [DragStartBehavior.down], [onStart] will be called at the time and position when a down event was first detected.

statement is probably wrong. https://github.com/flutter/flutter/pull/26246/files

Not sure if we should fix the behavior or the documentation.